onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java

```diff
@@ -139,6 +139,19 @@ public static void start(Context context,
     }
 
     /**
+     * Starts the MapActivity with a particular stop focused with the center of
+     * the map at a particular point.
```

Comment:
@cagryInside I think this comment should be changed to reflect the lack of location?